test/puppeteer.spec.js

```diff
@@ -264,6 +264,18 @@ module.exports.addTests = function({testRunner, expect, PROJECT_ROOT, defaultBro
         await page.close();
         await browser.close();
       });
+      it('should have default url when launching browser', async function() {
+        const browser = await puppeteer.launch();
```

Comment:
Please use the default browsing arguments here. See other tests that launch chrome.

Reply:
yes, I figured this out. pushed a fix. 10x!